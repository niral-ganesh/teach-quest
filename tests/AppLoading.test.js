import React from "react";
import { render } from "@testing-library/react-native";
import AppLoading from "../app/components/AppLoading";

it("renders correctly", () => {
  const tree = render(<AppLoading />);
  expect(tree).toMatchSnapshot();
});
