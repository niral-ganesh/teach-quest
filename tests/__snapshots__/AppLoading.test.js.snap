// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "alignItem": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    color="#0077b6"
  />
</View>
`;
